dockers/docker-router-advertiser/docker-router-advertiser.supervisord.conf.j2

```diff
@@ -30,7 +30,7 @@ dependent_startup=true
 {# Router advertiser should only run on ToR (T0) devices which have #}
 {# at least one VLAN interface which has an IPv6 address asigned #}
 {%- set vlan_v6 = namespace(count=0) -%}
-{%- if DEVICE_METADATA.localhost.type == "ToRRouter" -%}
+{%- if "ToRRouter" in DEVICE_METADATA.localhost.type -%}
```

Comment:
this seems to me some behaviorial change if the device is MgmtTorRouter. I am not saying this is not a good change, but maybe you want to confirm with @sumukhatv on M0 device.